packages/phone-number-privacy/src/salt-generation/query-quota.ts

```diff
@@ -0,0 +1,56 @@
+import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
+import fetch from 'node-fetch'
+import config from '../config'
+import { getPerformedQueryCount } from '../database/wrappers/account'
+import { getContractKit } from '../web3/contracts'
+
+const NUM_ATTESTATIONS_REQUIRED = 3
+const contractKit = getContractKit()
+
+/*
+ * Returns how many queries the account can make based on the
+ * calculated query quota and the number of queries already performed.
+ */
+export async function getRemainingQueryCount(account: string, phoneNumber: string) {
+  const queryQuota = await getQueryQuota(account, phoneNumber)
+  const performedQueryCount = await getPerformedQueryCount(account).catch((reason) => {
```

Comment:
Similar to index, lets not mix promises and async/await.
If you're having to add the .catch to make the linter happy it means something else is wrong. `await` should already unwrap the promise

Reply:
i was originally part of the db request handling .then()/.catch() -- but I didn't want to handle it in wrapper of the account table itself since I felt error handling for the db request should be handled on a per-case basis.  what we want to do when there is a db error is circumstantial.

here it catches it and allows the transaction